Makefile

```diff
@@ -1,7 +1,7 @@
 # Copyright 2024 Chainguard, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-# BEGIN: lint-install ../bincapz
+# BEGIN: lint-install ../malcontent
```

Comment:
Also dependent on renaming the repository (though anyone can rename their local directory).